src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -549,6 +563,43 @@ Apply to thead to make headers not bold.
 }
 
 /*////////////////////////////////////////////////////////////
+/////////////////////// Static page /////////////////////////
+//////////////////////////////////////////////////////////*/
+.navbar-brand {
+    padding: 0;
+}
+
+.caption {
+    color: #e46d0b;
+}
+
+.subcaption {
```

Comment:
Since these are only now color definitions, can we give them a color name instead?